packages/server-wallet/src/wallet/store.ts

```diff
@@ -19,14 +18,15 @@ import {Channel, SyncState, RequiredColumns} from '../models/channel';
 import {SigningWallet} from '../models/signing-wallet';
 import {addHash} from '../state-utils';
 import {ChannelState} from '../protocols/state';
+import knex from '../db/connection';
 
 export const Store = {
```

Comment:
As a minor comment, we should perhaps rename this `Store` to `PostgreSQLStore` if we're trying to be deliberate about defining and reusing this abstraction in other contexts like the browser.